README.rst

```diff
@@ -34,6 +34,116 @@ Dependency Management Toolkit for Google Cloud Python Projects
 .. |self_compatibility| image:: http://35.226.8.89/self_compatibility_badge/image?package=compatibility_lib
    :target: http://35.226.8.89/self_compatibility_badge/target?package=compatibility_lib
 
+Noogler Quickstart Guide (gLinux)
```

Comment:
Change the section name to `Development workflow`, since this repo is for external world not for Googlers :)